lib/monkey_patches.rb

```diff
@@ -0,0 +1,40 @@
+module Git
+  class Diff
+    # Monkey patch process_full_diff until https://github.com/schacon/ruby-git/issues/326 is resolved
+    def process_full_diff
+      defaults = {
+        :mode => '',
+        :src => '',
+        :dst => '',
+        :type => 'modified'
+      }
+      final = {}
+      current_file = nil
+      full_diff_utf8_encoded = @full_diff.encode("UTF-8", "binary", {
+    :invalid => :replace,
+    :undef => :replace
+  })
```

Comment:
Yeah...That was a copy-n-paste from the git ruby gem.  It has a combo of spaces and tabs :-(